apps/server/lib/lexical/server/code_intelligence/completion/env.ex

```diff
@@ -475,6 +570,28 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Env do
     end
   end
 
+  @token_types ~w(alias)a
+
+  def previous_token_column(prefix, token_type) when token_type in @token_types do
+    {_, _, _, _, tokens} = prefix |> String.to_charlist() |> :elixir_tokenizer.tokenize(1, 1, [])
+    prefix_tokens = Enum.reverse(tokens)
+
+    column =
+      Enum.find_value(prefix_tokens, fn
+        {:identifier, {_, column, _}, :__MODULE__} -> column
+        {:alias, {_, column, _}, _} -> column
+        _ -> nil
+      end)
+
+    # else
```

Comment:
Should we need to make this function more dynamic? not only search the alias token column?